apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -245,12 +245,15 @@ export class EmailOutputRendererUsecase {
     });
   }
 
-  private stringToBoolean(value: unknown): boolean {
-    if (typeof value === 'string') {
-      return value.toLowerCase() === 'true';
-    }
+  private stringToBoolean(value: string): boolean {
+    const normalized = value.toLowerCase().trim();
+    if (normalized === 'false' || normalized === 'null' || normalized === 'undefined') return false;
 
-    return false;
+    try {
+      return Boolean(JSON.parse(value));
```

Comment:
what is the usecase for executing JSON.parse?

Reply:
If I would remove the `JSON.parse` and just do `Boolean(value)`, some originally falsy values (like zeroes in the example) would resolve to `true` because it would be treated a string, but they should be `false`.

```
Boolean('0') // returns true (incorrect: because `0` as a number is falsy)
Boolean(JSON.parse('0')) // returns false

Boolean('0e0') // returns true
Boolean(JSON.parse('0e0')) // returns false

Boolean('-0') // returns true
Boolean(JSON.parse('-0')) // returns false
...
```

The issue we currently have is that the liquid parser always returns string, we can't distinguish if user typed `100` or `"100"` in the payload, we always assume it was number for now.

Reply:
shouldn't we also pass here the `normalized` value?